docs/guides/en/install-extensions.md

```diff
@@ -1,52 +1,54 @@
-## Installing additional extensions
+# Installing additional extensions
 
-It is recommended to install additional PHP extensions with https://github.com/mlocati/docker-php-extension-installer
+It is recommended to install additional PHP extensions with <https://github.com/mlocati/docker-php-extension-installer>
```

Comment:
`<>` not need

Reply:
I'm following Markdown rules. Do I make your change or follow the rules?
[](https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md034.md)